src/components/BlogPost.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { ListGroupItem, Collapse } from 'reactstrap';
+
+class BlogPost extends React.Component {
+  constructor(props) {
+    super(props);
+    
+    this.toggle = this.toggle.bind(this);
+    this.state = {collapse: false};
+  }
+  
+  toggle() {
+    this.setState({ collapse: !this.state.collapse });
+  }
```

Comment:
`bind` is a bit fragile, if you just do `toggle = () => {` you get the same result and less lines

Reply:
I don't understand, what lines do I need to change?

Reply:
Ditch line 8, change like 12 to the commented code, should work the same.

Reply:
I think we don't have that transform (class-props or whatever it's called) turned on in babel, unless it's changed what is on by default since the last time i checked. But we certainly could do it